test/integration/ads_integration.cc

```diff
@@ -46,6 +46,17 @@ envoy::api::v2::Cluster AdsIntegrationTest::buildCluster(const std::string& name
                                                                      name));
 }
 
+envoy::api::v2::Cluster AdsIntegrationTest::buildRedisCluster(const std::string& name) {
+  return TestUtility::parseYaml<envoy::api::v2::Cluster>(fmt::format(R"EOF(
+      name: {}
+      connect_timeout: 5s
+      type: EDS
+      eds_cluster_config: {{ eds_config: {{ ads: {{}} }} }}
+      lb_policy: MAGLEV
+    )EOF",
+                                                                     name));
```

Comment:
formatting seems off here

Reply:
It is same as buildCluster method, I thought it is some kept it like for some reason. Let me try formatting both.

Reply:
If I change this and reformat, it is coming like this only. So may be formatter is like this.,

Reply:
ok, that's fine